samples/main.go

```diff
@@ -184,16 +188,38 @@ func main() {
 			fmt.Printf("\n\n")
 		} else if choiceN == 3 {
 			fmt.Println("You selected choice to list snapshot from volume")
-			fmt.Printf("Please enter volume ID to get the snapshots: ")
-			_, _ = fmt.Scanf("%s", &volumeID)
-			fmt.Printf("\n")
-			snapshots, errr := sess.ListAllSnapshots(volumeID)
-			if errr == nil {
-				ctxLogger.Info("Successfully get snapshot details ================>", zap.Reflect("Snapshot ID", volumeID))
-				ctxLogger.Info("List of snapshots ", zap.Reflect("Snapshots are->", snapshots))
-			} else {
-				errr = updateRequestID(errr, requestID)
-				ctxLogger.Info("Failed to get snapshot details ================>", zap.Reflect("Snapshot ID", volumeID), zap.Reflect("Error", errr))
+			tags := map[string]string{}
```

Comment:
Is sample program working now with CI ?

Reply:
https://github.ibm.com/alchemy-containers/armada-storage/issues/3801
Needs to tested and fixed